pages/index.js

```diff
@@ -37,9 +54,21 @@ function Home(props) {
           Search for a Holiday!
         </h1>
         <h2>Countries</h2>
-        <select>
-        {countries.map(country => <option key={country.code}>{country.name}</option>)}
+        <select onChange={e => setCountry(e.target.value)}>
```

Comment:
`onChange` is not needed. use `ref` https://reactjs.org/docs/refs-and-the-dom.html

Reply:
> When to Use Refs
> There are a few good use cases for refs:
> - Managing focus, text selection, or media playback.
> - Triggering imperative animations.
> - Integrating with third-party DOM libraries.
> 
> Avoid using refs for anything that can be done declaratively.

Reply:
What is flawed in onChange over ref ?

Reply:
No. it is not flawed. I want you to achieve same thing using `refs` and not `onChange` event.
you can get more idea about it from here https://reactjs.org/docs/refs-and-the-dom.html#accessing-refs

Reply:
They have mentioned there that

>You may not use the ref attribute on function components because they don’t have instances.